tests/unit/tasks/fixSourceMaps.ts

```diff
@@ -1,5 +1,6 @@
-import * as registerSuite from 'intern!object';
-import * as assert from 'intern/chai!assert';
+const { registerSuite } = intern.getInterface('object');
```

Comment:
I might be noobing out, but do we need this anymore? I thought the `fixSourceMaps` task was only because of intern 3.

Reply:
You might be right. On these tasks, I did a 1:1 conversion without thinking if they needed removed.

Reply:
It turns out this task is to remove the path in the inline source maps rather than remapping the source maps, so it needs to stay